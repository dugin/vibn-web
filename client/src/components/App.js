import React, {Component} from 'react';
import './App.css';
import {Route, Switch, Redirect} from 'react-router-dom';
import RegisterPage from "./register/Register.page";

class App extends Component {

    render() {
        return (
            <div className="App">

                <Switch>
                    {/*<Route exact path="/admin/signin" component={LoginPage}/>*/}
                    <Route exact path="/admin/events/register/" component={RegisterPage}/>
                    <Redirect from="**" to="/admin/events/register/"/>
                </Switch>
            </div>
        );
    }
}

export default App;
