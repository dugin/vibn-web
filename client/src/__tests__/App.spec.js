import React from 'react';
import {shallow} from 'enzyme';
import toJson from 'enzyme-to-json';
import App from "../components/App";
import './../setupTests';

describe('App', () => {
    const app = shallow(<App/>);

    it('renders properly', () => {
        expect(toJson(app)).toMatchSnapshot();
    });


});