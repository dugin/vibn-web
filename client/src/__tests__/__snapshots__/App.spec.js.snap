// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders properly 1`] = `
<div
  className="App"
>
  <Switch>
    <Route
      component={[Function]}
      exact={true}
      path="/admin/events/register/"
    />
    <Redirect
      from="**"
      push={false}
      to="/admin/events/register/"
    />
  </Switch>
</div>
`;
